tensorflow/lite/experimental/micro/examples/micro_speech/Makefile.inc

```diff
@@ -112,6 +113,9 @@ tensorflow/lite/experimental/micro/examples/micro_speech/micro_features/filterba
 tensorflow/lite/experimental/micro/examples/micro_speech/micro_features/filterbank_util.cc
 
 MICRO_FEATURES_FILTERBANK_TEST_HDRS := \
+tensorflow/lite/experimental/micro/examples/micro_speech/micro_features/bits.h \
+tensorflow/lite/experimental/micro/examples/micro_speech/micro_features/fft.h \
+tensorflow/lite/experimental/micro/examples/micro_speech/micro_features/micro_model_settings.h \
```

Comment:
micro_model_settings.h should be moved so that the list is in alpabetical order.